src/main.cpp

```diff
@@ -88,6 +88,8 @@ AGEHook<0x6B16A4>::Type<char[40]> cityName2;
 
 AGEHook<0x62B068>::Type<int> timeOfDay;
 
+AGEHook<0x627518>::Type<int> carLights;
```

Comment:
Looks like you found `vehCar::bHeadlights`, maybe change the name `bHeadlights`?